data/schedules_report/daily.yaml

```diff
@@ -0,0 +1,18 @@
+name: testing schedule 1
+description: description of testing schedule 1
+filter:
+    - Events
+    - Policy
+    - Policy Events for Last Week
+active: True
+timer:
+    run: Daily
+    hours: 2 Days
```

Comment:
Question : why is it run in 2 days ? can it be run in 1 day ?

Reply:
@sshveta I will change it to 1 day.